src/App/App.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import GameplayScreen from 'Screens/GameplayScreen/GameplayScreen';
+import { isBrowserValid, invalidBrowserMessage } from './AppUtils';
+
+export default function App() {
+  const isBrowserValidBool = isBrowserValid();
+  if (isBrowserValidBool) {
```

Comment:
all this logic should all be a function, App.js shouldn't know the mechanics of the valid-browser check and what message to display.